### ISSUE TITLE
Last commits broke our deployments through GitHub Actions

### ISSUE THREAD

**Issue by Vichoko** [google-github-actions/setup-gcloud#339]
### TL;DR
One of the last 2 commits on master broke our GitHub Actions. I guess it has to be related to [this commit.](https://github.com/google-github-actions/setup-gcloud/commit/38fab8887c8defeb67814b4ccf61b916ce8b9220)

**Expected behavior**
This error started to pop up an hour ago, without doing changes to my deployment files.

**Observed behavior**
```
Download action repository 'GoogleCloudPlatform/github-actions@master'
Error: Can't find 'action.yml', 'action.yaml' or 'Dockerfile' under '/home/runner/work/_actions/GoogleCloudPlatform/github-actions/master/setup-gcloud'. Did you forget to run actions/checkout before running your local action?
```

The temporary solution involved fixing the action to the last working commit:
```
      - name: Setup gcloud
        uses: GoogleCloudPlatform/github-actions/setup-gcloud@7159039b8c614f8ed6d9bd75af3f6113681a8ff4

```

**Additional information**

**Comment by bharathkkb** [comment]
Hi @Vichoko 
This was intended as part of some cleanup to reduce technical debt. Please follow https://github.com/google-github-actions/setup-gcloud#-notice

**Comment by bshevlin** [comment]
@bharathkkb To confirm, every build action using `setup-gcloud` has to update their references or they will not work going forward? Or are there plans to make the change backwards-compatible?

**Comment by bharathkkb** [comment]
@bshevlin correct, every reference using `GoogleCloudPlatform/github-actions/setup-gcloud@master` will need to be updated to `google-github-actions/setup-gcloud@master`. 

We were in backwards compatibility mode for the past ~6ish months since we switched orgs with a [deprecation notice](https://github.com/google-github-actions/setup-gcloud/tree/7159039b8c614f8ed6d9bd75af3f6113681a8ff4/setup-gcloud) and [surfacing](https://github.com/google-github-actions/setup-gcloud/blob/7159039b8c614f8ed6d9bd75af3f6113681a8ff4/setup-gcloud/dist-frozen/index.js#L16353) it in the action logs  so hoping it isn't too painful to swap out.

**Comment by Vichoko** [comment]
Thanks for the clarification! 

Just want to comment that I think the transition would have been smoother (we wouldn't notice this change so catastrophically) if this feature were instead thrown into a stable release. I mean, the newest release was in March, so following `master` was natural. On the other hand, freezing the action into a commit hash just doesn't look so elegant as a release would be.

**Comment by bharathkkb** [comment]
@Vichoko thanks for the feedback! We usually cut releases when we have new features/fixes (pinned actions on `0.2` or `0.2.x` should still work). Previously `GoogleCloudPlatform/github-actions/setup-gcloud@master` pointed to a prebuilt binary for compat. We eventually wanted users who were using the old version to use the newer versions of the binary which will contain newer bug/dependency fixes.
